configs/deeplabv3/deeplabv3_r101-d8_512x1024_160k_a2d2.py

```diff
@@ -0,0 +1,2 @@
+_base_ = './deeplabv3_r50-d8_512x1024_160k_a2d2.py'
```

Comment:
512x2014 --> 480x1208 in config name.

Reply:
Corrected resolution 512x1024 ==> 480x1208